applications/search/src/api/apis.js

```diff
@@ -1,12 +1,13 @@
 import axios from 'axios';
+import { getConfig } from '../config';
 
-export const apisUrlBase = '/api/apis';
+export const apisUrlBase = () => `${getConfig().apiApi.host}/api/apis`;
 
-export const apisSearch = params => axios(apisUrlBase, { params }).then(r => r.data);
+export const apisSearch = params => axios(apisUrlBase(), { params }).then(r => r.data);
```

Comment:
Do we want to catch the result as well?

Reply:
jeg tenker at klienten skulle gjøre catch, men jeg har ikke sjekket alle sted hvor det kalles... skal det nå